groups/alteryx-live/data.tf

```diff
@@ -31,6 +31,40 @@ data "vault_generic_secret" "azure_dc_cidrs" {
   path = "applications/${var.aws_account}-${var.aws_region}/${var.application}/azure_dc"
 }
 
+data "vault_generic_secret" "vpc" {
```

Comment:
I would combine all these settings into a single entry within Vault. There's no need to keep them separate